src/System.Net.Security/src/System/Net/SslStreamPal.Windows.cs

```diff
@@ -53,7 +53,7 @@ public static SecurityStatusPal AcceptSecurityContext(ref SafeFreeCredentials cr
                 outputBuffer,
                 ref unusedAttributes);
 
-            return GetSecurityStatusPalFromWin32Int(errorCode);
+            return SecurityStatusAdapterPal.GetSecurityStatusPalFromWin32Int(errorCode);
```

Comment:
It's a bit strange to have a method on a ```Pal``` class that includes a platform-specific name.  I'd have expected a more generic ```GetSecurityStatusPalFromNativeError``` or something that could be named the same across platforms.  No need to block on this, but we should revisit whether this is the right factoring.